scripts/pylib/twister/twisterlib/harness.py

```diff
@@ -324,12 +325,18 @@ def generate_command(self):
         ]
         command.extend([os.path.normpath(os.path.join(
             self.source_dir, os.path.expanduser(os.path.expandvars(src)))) for src in pytest_root])
-        command.extend(pytest_args)
+
+        if handler.options.pytest_args:
+            command.append(handler.options.pytest_args)
```

Comment:
```suggestion
            command.extend(handler.options.pytest_args)
```
should it `extend` ?

Reply:
No, `append`, because CLI is as a string and pytest_args in yaml is given as a list of strings